interface.go

```diff
@@ -16,33 +17,49 @@ type TableInfo struct {
 }
 
 type ColumnDef struct {
-	Path string    `json:"path"`
+	Name string    `json:"name"`
+	Key  string    `json:"key"`
 	Type FieldType `json:"type"`
 }
 
 type TableDriver interface {
 	New(name string, columns []ColumnDef) (TableStore, error)
 	Get(name string) (TableStore, error)
+	GetAllColNames() chan string
+	GetLabels(edges bool) chan string
```

Comment:
What does this method do?

Reply:
GetLabels gets all of the vertex labels of all of the edge labels. This is used for list labels functions. See https://github.com/bmeg/grip/blob/feature/grids/grids/graph.go#L921-L936 in grip. 

GetAllColNames was designed to aggregated row level information on all tables for index operations, but it didn't end up getting used in this PR.